ci-operator/step-registry/upi/deprovision/aws/check-destroy/upi-deprovision-aws-check-destroy-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I believe this behavior should be part of `upi-deprovision-aws.sh` instead of having to duplicate the effort here. I expect the reason it is here is because there are things that need to happen between the first call of `upi-deprovision-aws.sh` and when this is actually called. I believe `upi-aws-post-chain` can be changed to remove this.

Reply:
We had a problem with some kind of cyclic dependency error where the vpc and the security groups associated with the vpc were not getting deleted. I will try to run it one more time to see if I face that error and replace this if necessary